tasks/config.yml

```diff
@@ -11,7 +11,7 @@
     backup: true
```

Comment:
..so not 100% sure I understand the directory structure refactor here. I mean, 'common' was meant to allow an easy/advertised path to supporting multiple operating/distribution system types. I can see forming a precedent that placing files in the main directory marks them as "common or main" files though seems kind of arbitrary.

Reply:
Maybe I was to rude, I have no good experience of using sub directories. The role is not extremely complex. In similar roles that I create, I usually name the files according to the function. I'll be happy to restore the folder structure if you want to.

Question: is it common that there are sub folders in roles? I cannot find that many examples.

Reply:
Ha no worries and it isn't at all. Definitely has to be the simplest of all the roles at least under 0xO1. As for sub folders, I'm generally just a stickler for organization but can't say I'm familiar with any sort of official convention either way.